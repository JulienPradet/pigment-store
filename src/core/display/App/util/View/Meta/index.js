import React from 'react'
import styles from './meta.m.css'

const Meta = ({children}) => <div className={styles.meta}>{children}</div>

export default Meta
