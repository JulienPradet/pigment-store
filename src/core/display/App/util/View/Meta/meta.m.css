.meta {
  font-size: 0.9em;
  color: #999;
}
