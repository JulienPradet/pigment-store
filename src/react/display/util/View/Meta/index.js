import React from 'react'
import styles from './meta.css'

export const Meta = ({children}) => <div className={styles.meta}>{children}</div>
