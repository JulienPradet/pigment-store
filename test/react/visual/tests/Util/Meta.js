import React from 'react'
import PigmentStore from '../../../../../dist/index'
import Meta from '../../../../../src/react/display/util/View/Meta/index'

export default PigmentStore.React.describe('Meta', Meta)
  .featureJsx(
    'Default',
    <Meta>Faded text</Meta>
  )
