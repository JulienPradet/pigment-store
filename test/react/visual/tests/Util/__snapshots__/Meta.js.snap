exports[`Meta Default 1`] = `
<div
  className="meta">
  Faded text
</div>
`;
